ext/websocket/01_websocket.js

```diff
@@ -54,13 +54,15 @@ const {
   op_ws_send_text,
   op_ws_next_event,
   op_ws_send_ping,
+  op_ws_get_buffered_amount,
 } = core.generateAsyncOpHandler(
   "op_ws_create",
   "op_ws_close",
   "op_ws_send_binary",
   "op_ws_send_text",
   "op_ws_next_event",
   "op_ws_send_ping",
+  "op_ws_get_buffered_amount",
```

Comment:
This is suspicious - `op_ws_get_buffered_amount` appears to be a sync op - why is it generated here?

Reply:
`generateAsyncOpHandler` is a misnomer and generates sync ops too. It used to be `ensureFastOps` (https://github.com/denoland/deno/pull/18887/commits/6a5c193126e95a271e296048c586f56e798ab3a4) but we renamed it. 

It's turned out to be a convenient (though repetitive way) to initialize op functions that I wonder if we should go back and rename it.

Reply:
Okay, that makes sense. I think we missed some of the ops that should use it because of the name. Let's rename it to `ensureFastOps` 👍

Reply:
Follow-up in https://github.com/denoland/deno/pull/19377